ArcGISRuntimeSDKQt_CppSamples/Layers/AddDynamicEntityLayer/AddDynamicEntityLayer.qml

```diff
@@ -17,6 +17,7 @@
 import QtQuick
 import QtQuick.Controls
 import Esri.Samples
+import Esri.ArcGISRuntime.Toolkit
```

Comment:
since we are adding the toolkit dependency, we need add the toolkit includes into the .pro file

Reply:
added dynamic entity path.